shotgun_api3/lib/README.md

```diff
@@ -0,0 +1,52 @@
+# Lib Submodules
+
+## Third Party Modules
+
+Some third-party modules are bundled with `python-api` inside lib.
+
+### httplib2
+
+`httplib2` is used to make http connections to the shotgun server.  We bundle both python2 and python3 compatible versions since httplib2 chose to maintain parallel versions of the module for python 2 and 3 compatibility.
+
+The version of `httplib2` bundled should be updated manually, however its version is included in the unused `shotgun_api3/lib/requirements.txt` to allow Github's automated CVE notifications to work.
```

Comment:
should we explain a bit more about this file? Seems like we are mentioning it just in passing - maybe we should advertise it a bit more (own section) to make sure devs understand to update the versions there when they upgrade libraries?

Reply:
Added a section explaining the purpose of the file and highlighting the importance of keeping it updated.